configs/relengworker-prod/config.yml

```diff
@@ -701,6 +701,34 @@ worker_types:
         capacity_ratio: 1.0
         min_replicas: 0
 
+  - worker_type: mobile-1-tree
+    provisioner: scriptworker-k8s
+    root_url: "https://firefox-ci-tc.services.mozilla.com"
+    deployment_namespace: prod-tree
+    deployment_name: tree-prod-relengworker-firefoxci-mobile-1-1
+    autoscale:
+      algorithm: slo
+      args:
+        max_replicas: 1
+        avg_task_duration: 60
+        slo_seconds: 120
+        capacity_ratio: 1.0
+        min_replicas: 0
+
+  - worker_type: mobile-3-tree
+    provisioner: scriptworker-k8s
+    root_url: "https://firefox-ci-tc.services.mozilla.com"
+    deployment_namespace: prod-tree
+    deployment_name: tree-prod-relengworker-firefoxci-mobile-3-1
+    autoscale:
+      algorithm: slo
+      args:
+        max_replicas: 3
```

Comment:
I'm curious as to why this is set to `3` since we're expecting only Fenix releases to use it for now. Certainly not blocking.

Reply:
I just copied what we had in gecko-tree-3, but I agree: 1 is good for now.